staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -198,6 +197,7 @@ func newWatchCache(
 	getAttrsFunc func(runtime.Object) (labels.Set, fields.Set, error),
 	versioner storage.Versioner,
 	indexers *cache.Indexers,
+	clock clock.Clock,
```

Comment:
Can you please move the clock changes (and the restore of the test behavior) to a separate PR?

Reply:
Done, see [#92174](https://github.com/kubernetes/kubernetes/pull/92174)